app/src/main/res/layout/weight_record_fragment.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout>
+    <data>
+
+        <variable
+            name="weightRecordViewModel"
+            type="com.example.weightcontrolapp.ui.weightrecord.WeightRecordViewModel" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <TextView
+            android:id="@+id/textview_record_your_weight"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/record_your_weight"
+            android:textSize="30sp"
+            app:layout_constraintBottom_toTopOf="@id/weight_record_button"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <EditText
```

Comment:
You should Add the unit of weight next to this EditText.

Reply:
[ee17e5c](https://github.com/kenta7777/WeightControlApp/pull/1/commits/ee17e5cfdc52e44e453fbb26e6cc82fd2a3552db)